expertise/models/specter2_scincl/specter.py

```diff
@@ -68,6 +68,11 @@ def __init__(self, specter_dir, work_dir, average_score=False, max_score=True, b
         self.venue_specific_weights = venue_specific_weights
         self.normalize_scores = normalize_scores
         print(f"SPECTER2 venue_specific_weights: {venue_specific_weights}")
+        self.embeddings_cache = embeddings_cache
+        if self.embeddings_cache is not None and self.embeddings_cache.is_connected:
+            self.use_cache = True
```

Comment:
if `use_cache` a setting of the server? or something that we can set in the expertise request call?

Reply:
We can keep it as it is for now.